src/Traits/DoctrineEncrypt.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Ambta\DoctrineEncryptBundle\Traits;
+
+use Ambta\DoctrineEncryptBundle\Service\Encrypt;
+
+trait DoctrineEncrypt
```

Comment:
Same here as with the service, and to have the name more clear and similar to other classes like this, I propose renaming this to `EncryptServiceAwareTrait`.

This could be accompanied by a `EcryptServiceAwareInterface`, which you could use to have the symfony container automatically inject the `EncryptService` to. 

This would as a bonus allow any developers creating classes which get the service automatically injected as well.